uflash.py

```diff
@@ -689,26 +601,22 @@ def main(argv=None):
     else:
         try:
             flash(path_to_python=args.source, paths_to_microbits=args.target,
-                  path_to_runtime=args.runtime, minify=args.minify,
                   keepname=False)
         except Exception as ex:
-            error_message = (
-                "Error flashing {source} to {target}{runtime}: {error!s}"
-            )
+            error_message = ("Error flashing {source} to {target}: {error!s}")
             source = args.source
             target = args.target if args.target else "microbit"
-            if args.runtime:
-                runtime = "with runtime {runtime}".format(runtime=args.runtime)
-            else:
-                runtime = ""
-            print(error_message.format(source=source, target=target,
-                                       runtime=runtime, error=ex),
+            print(error_message.format(source=source, target=target, error=ex),
                   file=sys.stderr)
             sys.exit(1)
 
 
 #: A string representation of the MicroPython runtime hex.
-_RUNTIME = """:020000040000FA
+# This is a temporary hack to avoid changing the inline hex until the PR is
+# ready to be merged, otherwise the diff is too large to render in GitHub
+with open('upy-uh.hex', 'r') as runtime_file:
+    _RUNTIME = runtime_file.read()
+_OLD_RUNTIME = """:020000040000FA
```

Comment:
This is only added here so that the GitHub PR viewer can render the diff, if the hex file is updated GH will then say the diff is to large to be shown.

Once we are happy with the PR I'll add the last commit with the new Universal Hex file in `_RUNTIME`.